doc/old_and_new_proposal.md

```diff
@@ -644,6 +768,10 @@ calculated. If YaST is not able to make space for all the volumes required by
 those default settings, it will perform new attempts altering the settings. For
 that, it will follow the `disable_order` for each volume with that field.
 
+> FIXME: unclear (and not yet implemented anyway)
+>
+> `adjust_by_ram` is optional == `adjust_by_ram_configurable`?
+
```

Comment:
Yes, optional here stands for configurable. This is implemented in the PR that is being reviewed by @imobachgs : https://github.com/yast/yast-storage-ng/pull/384